lms/views/helpers/_via.py

```diff
@@ -18,9 +19,10 @@ def via_url(request, document_url, content_type=None):
     :param content_type: Either "pdf" or "html" if known, None if not
     :return: A URL string
     """
-
     doc = ViaDoc(document_url, content_type)
 
     return ViaClient(
-        service_url=request.registry.settings["via_url"], host_url=request.host_url
+        request.registry.settings["via_url"],
+        request.host_url,
+        ViaSecureURLToken(request.registry.settings["via_secret"]),
```

Comment:
My suggestions in https://github.com/hypothesis/h-vialib/pull/9/files would mean you don't have to import the `ViaSecureURLToken`, that would be an internal detail of the client. We'd just pass the secret here.